content/docs/command-reference/config.md

```diff
@@ -27,15 +27,27 @@ This command reads and updates the DVC configuration files. By default, the
 regular project's config file in `.dvc/config` is read or modified. This file is
 meant to be tracked by Git and should not contain sensitive and/or user-specific
 information (passwords, SSH keys, etc). Use the `--local` command option (flag)
-instead, to set (or override) secrets. `--global` and `--system` are also
-available to set config options for multiple projects and users, respectively.
-
-| Command option | Priority | Linux location         | Windows location                                          |
-| -------------- | -------- | ---------------------- | --------------------------------------------------------- |
-| `--local`      | 1        | `.dvc/config.local`    | `.dvc/config.local`                                       |
-| None (default) | 2        | `.dvc/config`          | `.dvc/config`                                             |
-| `--global`     | 3        | `~/.config/dvc/config` | `%LocalAppData%/iterative/dvc/config`                     |
```

Comment:
should we use Windows style path for Windows?
should we put escape before " " in all of these string so that it's easier to copy/paste them?

Reply:
Good points. Applied.